.github/workflows/deploy_website.yaml

```diff
@@ -0,0 +1,82 @@
+name: Deploy Website
+
+on:
+  pull_request:
+    types:
+      - closed # Trigger when the PR is closed
+    branches:
+      - 'main' # Target branch is main
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'The environment used as target'
+        type: choice
+        required: true
+        default: prod
+        options:
+          - prod
+
+  # Allows external webhook trigger
+  repository_dispatch:
+    types:
+      - webhook
+
+# These permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+
+  cd_deploy:
+    name: Deploy to S3
+    # Trigger when the PR is merged
+    if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
+    runs-on: 'ubuntu-latest'
+    strategy:
+      matrix:
+        environment: [ 'prod' ]
+
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - name: Deploy website
+        uses: ./.github/actions/deploy
+        with:
+          environment: ${{ matrix.environment }}
+          path_to_gitbook_docs: ${{ github.workspace }}/apps/nextjs-website/docs
```

Comment:
The system does not need the `path_to_gitbook_docs`

Reply:
ok, change made with the latest commit